dmoj/cptbox/filesystem_policies.py

```diff
@@ -0,0 +1,117 @@
+import os
+from enum import Enum
+from typing import List, Union
+
+
+class AccessMode(Enum):
+    NONE = 0
+    EXACT = 1
+    RECURSIVE = 2
+
+    # flake8 doesn't recognize `AccessMode` in type annotations here
+    @classmethod
+    def more_permissive(cls, mode1, mode2):
+        return AccessMode(max(mode1.value, mode2.value))
```

Comment:
```suggestion
        return cls(max(mode1.value, mode2.value))
```
Is this more appropriate?

Reply:
I don't know. Maybe? Probably? I'll commit it anyway.

Reply:
Force pushed new commits that includes this change